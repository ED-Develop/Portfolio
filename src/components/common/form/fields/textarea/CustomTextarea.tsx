import React from 'react';
import style from '../../CustomFields.module.scss';
import {Input} from 'antd';
import {TextAreaProps} from 'antd/lib/input/TextArea';

export const CustomTextarea: React.FC<TextAreaProps> = (props) => {
    return <Input.TextArea {...props} className={`${style.field} ${style.field_textarea}`}/>
};