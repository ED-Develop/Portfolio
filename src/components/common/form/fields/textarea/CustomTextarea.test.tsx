import React from 'react';
import {render} from '@testing-library/react';
import {CustomTextarea} from './CustomTextarea';

describe('CustomTextarea component', () => {
    test('Should renders correctly', () => {
        const {container} = render(<CustomTextarea/>);

        expect(container).toMatchSnapshot();
    })
})