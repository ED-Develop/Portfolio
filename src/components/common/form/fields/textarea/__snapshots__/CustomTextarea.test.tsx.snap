// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTextarea component Should renders correctly 1`] = `
<div>
  <textarea
    class="ant-input field field_textarea"
  />
</div>
`;
